app/helpers/form_helper.py

```diff
@@ -0,0 +1,109 @@
+from app.helpers.schema_helper import SchemaHelper
+from app.forms.household_composition_form import generate_household_composition_form, deserialise_composition_answers
+from app.forms.household_relationship_form import build_relationship_choices, generate_relationship_form
+from app.forms.questionnaire_form import generate_form
+
+
+def get_form_for_location(block_json, location, answer_store, error_messages, disable_mandatory=False):
```

Comment:
We have had discussions on this before with the disable_mandatory flag. that flag gets passed through many functions to change a nested behaviour.